tntconcept-binnacle/src/main/kotlin/com/autentia/tnt/binnacle/core/domain/DateInterval.kt

```diff
@@ -17,6 +18,23 @@ data class DateInterval private constructor(val start: LocalDate, val end: Local
             LocalDate.of(year, Month.JANUARY, 1),
             LocalDate.of(year, Month.DECEMBER, 31)
         )
+
+        fun getDateIntervalForActivityList (
+            activities: List<Activity>,
+            defaultTimeInterval: TimeInterval,
+        ): DateInterval {
+
+            if (activities.isNotEmpty()) {
```

Comment:
Este código puede sustituirse por `return if() {} else{}

         fun getDateIntervalForActivityList (   
            activities: List<Activity>,
            defaultTimeInterval: TimeInterval,
        ): DateInterval {
            return if (activities.isNotEmpty()) {
                val minDate = activities.minOf { it.getStart() }
                val maxDate = activities.maxOf { it.getEnd() }
                of(minDate.toLocalDate(), maxDate.toLocalDate())
            } else {
                defaultTimeInterval.getDateInterval()
            }}